src/chunkserver/copyset_node.cpp

```diff
@@ -196,11 +197,13 @@ int CopysetNode::Run() {
     }
 
     if (!enableOdsyncWhenOpenChunkFile_) {
-        CHECK_EQ(0, syncTimer_.init(this, syncTimerIntervalMs_));
-        LOG(INFO) << "Init sync timer success, interval = "
-                  << syncTimerIntervalMs_;
+        dataStore_->SetCacheCondPtr(syncThread_.cond_);
+        dataStore_->SetCacheLimit(syncChunkLimit_);
+        syncThread_.Init(this, syncChunkLimit_);
```

Comment:
> The meaningful value of "syncThread_.cond_" will not be passed to the condition variable "cvar_" of each chunk only if "syncThread_" was initialized before datastore, but actually it wasn't. And, the statement "syncThread_.Init" should be run before "dataStore_->SetCacheCondPtr", or it will make nonsense.

thanks, fix. 🤣

Reply:
The second problem was fixed, but the first one still exists. Maybe you need to put the initialization of "syncThread" into the block of "CopysetNode::Init",  just before "datastore" initialization. 
![image](https://user-images.githubusercontent.com/19409549/194840155-e0fe7725-5a1d-4a16-85e0-c0f185dcc300.png)

Reply:
> The second problem was fixed, but the first one still exists. Maybe you need to put the initialization of "syncThread" into the block of "CopysetNode::Init", just before "datastore" initialization. ![image](https://user-images.githubusercontent.com/19409549/194840155-e0fe7725-5a1d-4a16-85e0-c0f185dcc300.png)



> The second problem was fixed, but the first one still exists. Maybe you need to put the initialization of "syncThread" into the block of "CopysetNode::Init", just before "datastore" initialization. ![image](https://user-images.githubusercontent.com/19409549/194840155-e0fe7725-5a1d-4a16-85e0-c0f185dcc300.png)

fix.